src/modules/rewards/hooks/useRewardsAuth.tsx

```diff
@@ -117,14 +127,16 @@ const useRewardsAuth = () => {
     caip10WalletAddress,
     status,
     unlockProfile,
-    showConnectModal,
-    setShowConnectModal,
+    isAuthModalVisible,
+    setIsAuthModalVisible,
```

Comment:
setIsAuthModalVisible

We don't need setIsAuthModalVisible Since we already export hideAuthModal,
    showAuthModal.